sdk/src/main/java/com/karhoo/sdk/analytics/Payloader.kt

```diff
@@ -227,5 +404,19 @@ class Payloader internal constructor(builder: Builder) {
         private const val PAYMENT_CARD_LAST_FOUR_DIGITS = "card_last_4_digits"
         private const val PAYMENT_AMOUNT = "amount"
         private const val PAYMENT_CURRENCY = "currency"
+
+        private const val LOYALTY_ENABLED = "loyalty_enabled"
+        private const val LOYALTY_NAME = "loyalty_name"
+        private const val LOYALTY_STATUS_SUCCESS = "loyalty_status_success"
+        private const val LOYALTY_STATUS_CAN_EARN = "loyalty_status_can_earn"
+        private const val LOYALTY_STATUS_CAN_BURN = "loyalty_status_can_burn"
+        private const val LOYALTY_STATUS_BALANCE = "loyalty_status_balance"
+        private const val LOYALTY_STATUS_ERROR_SLUG = "loyalty_status_error_slug"
+        private const val LOYALTY_STATUS_ERROR_MESSAGE = "loyalty_status_error_message"
```

Comment:
not used

Reply:
can't we use just "error_slug" and "error_message" for both status and preauth? And  "error_message" also for payment?

Reply:
we can. don't see why not. we should get enough context from the event name as to whether it is a slug related to loyalty status or loyalty pre-auth for example.